ext/oci8/oci8.stub.php

```diff
@@ -2,534 +2,468 @@
 
 /** @generate-function-entries */
 
-/** @param resource $statement_resource */
-function oci_define_by_name($statement_resource, string $column_name, mixed &$variable, int $type = 0): bool {}
+/** @param resource $statement */
+function oci_define_by_name($statement, string $column, mixed &$result, int $type = 0): bool {}
```

Comment:
```suggestion
function oci_define_by_name($statement, string $column, mixed &$var, int $type = 0): bool {}
```
In line with what we do elsewhere. If I understood right, oci_define_by_name is a bindColumn() analogon.